### PR TITLE
Merge 'gql-notifications' into 'dev'

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-171.surge.sh

Updated pages:
- [Changes to status codes per Neo4j version](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/changelogs)
- [List of all server error codes](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/errors/all-errors)
- [Server errors](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/errors)
- [Introduction](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5)
- [List of all server notification codes](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/notifications/all-notifications)
- [Server notifications](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/notifications)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-171.surge.sh

Updated pages:
- [Changes to status codes per Neo4j version](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/changelogs)
- [List of all server error codes](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/errors/all-errors)
- [Server errors](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/errors)
- [Introduction](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5)
- [List of all server notification codes](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/notifications/all-notifications)
- [Server notifications](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/notifications)

File: .github/workflows/docs-pr-checks.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ on:
     - "dev"
     - "5.x"
     - "4.[0-9]"
+    - "gql-notifications"
```
Comment (recrwplay) [inline .github/workflows/docs-pr-checks.yml:10]: Assuming we are going to delete this branch after merging, we don't need to include it here

File: .github/workflows/docs-teardown.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ on:
       - "dev"
       - "5.x"
       - "4.[0-9]"
+      - "gql-notifications"
```
Comment (recrwplay) [inline .github/workflows/docs-teardown.yml:10]: Same as for `docs-pr-checks.yml`. Assuming we are going to delete this branch after merging, we don't need to include it here

**Review by recrwplay — CHANGES_REQUESTED** [review]

File: .github/workflows/docs-pr-checks.yml (hunk 1 above)
Comment (NataliaIvakina) [inline .github/workflows/docs-pr-checks.yml:10]:
```suggestion
```

File: .github/workflows/docs-teardown.yml (hunk 2 above)
Comment (NataliaIvakina) [inline .github/workflows/docs-teardown.yml:10]:
```suggestion
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-171.surge.sh

Updated pages:
- [Changes to status codes per Neo4j version](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/changelogs)
- [List of all server error codes](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/errors/all-errors)
- [Server errors](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/errors)
- [Introduction](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5)
- [List of all server notification codes](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/notifications/all-notifications)
- [Server notifications](https://neo4j-docs-status-codes-171.surge.sh/status-codes/5/notifications)

**Review by recrwplay — APPROVED** [review]

**Pull request merged**